react/admin/settings/RMASettings.tsx

```diff
@@ -54,13 +85,18 @@ export const RMASettings = () => {
   const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault()
 
-    const { customReturnReasons, maxDays } = appSettings
+    const { customReturnReasons, maxDays, paymentOptions } = appSettings
+    const { allowedPaymentTypes, enablePaymentMethodSelection } = paymentOptions
 
     const maxCustomOptionsDays = customReturnReasons?.reduce(
       (maxDay, option) => (maxDay > option.maxDays ? maxDay : option.maxDays),
       0
     )
 
+    const isAnyPaymentMethod = Object.values(allowedPaymentTypes).some(
+      (paymentMethod) => paymentMethod === true
+    )
+
```

Comment:
The function you brought to this file can validate this and the if statement at once for you.